insecure/corruptnet/network.go

```diff
@@ -543,7 +543,7 @@ func (n *Network) HandleIncomingEvent(event interface{}, channel channels.Channe
 
 	err = n.attackerInboundStream.Send(msg)
 	if err != nil {
-		lg.Fatal().Err(err).Msg("could not send message to attack orchestrator to observe")
+		lg.Error().Err(err).Msg("could not send message to attack orchestrator to observe")
```

Comment:
should this return false?

Reply:
https://github.com/onflow/flow-go/pull/3580/commits/d4e9c4c5731603c6f0aa543e63ecf2059312d5a9